cellprofiler/modules/measureobjectsizeshape.py

```diff
@@ -367,6 +373,19 @@ def run_on_objects(self, object_name, workspace):
                 nobjects = 0
             else:
                 nobjects = numpy.max(objects.indices)
+
+            bounding_box_min_x = []
+            bounding_box_min_y = []
+            bounding_box_max_x = []
+            bounding_box_max_y = []
+
+            for label, index in objects.get_labels():
+                props = skimage.measure.regionprops(label)
+                bounding_box_min_x += [int(prop.bbox[0]) for prop in props]
```

Comment:
I think all you need to do here is set bounding_box_min_x to bbox[1], min_y to bbox[0], max_x to bbox[3], and max_y to bbox[2]